canonical/canonical.com
# Fix modal bug and apply final changes

## Done

- Added modal to /roadshow
- Added get-in-touch hash functionality 
- Removed requested blog post 
- Updated subnav MicroCloud logo

## QA

- View the site locally in your web browser at: https://canonical-com-1112.demos.haus/microcloud
- Click the "get in touch" CTA in the hero, see that the url has the #get-in-touch hash
- Visit https://canonical-com-1112.demos.haus/microcloud/roadshow
- At the bottom of the page, click the CTA and see that yoy are routed to the modal 


## Fixes

https://warthogs.atlassian.net/browse/WD-7226
